app/views/layouts/application.html.haml

```diff
@@ -6,6 +6,7 @@
 %html{lang: I18n.locale.to_s, dir: (rtl? ? "rtl" : "ltr")}
   %head{prefix: og_prefix}
     %meta{name: "viewport", content: "width=device-width, initial-scale=1"}/
+    %meta{name: "apple-itunes-app", content: "app-id=1538074832, app-argument=#{AppConfig.pod_uri}"}
```

Comment:
So @jhass said that the app doesn't use the URL to select the pod, but it needs the username (I assume in form of the diaspora ID? Or how else does the app know which pod to contact?), so I think we should also add the username part here (as parameter `?username=<username>`, because the apple documentation says this needs to be an URL?), so the app can parse the things it needs to directly redirect the user to the correct pod.

I must confess that I never used the app yet (I'm just more a desktop person), so I don't know exactly what the app asks for, but I just thought it would be cool to provide it with the needed information to make it easier for the user to log in the first time.

Reply:
Yes the app only asks for the diaspora ID

Reply:
@SuperTux88 I already added the username (aka diaspora ID) to the banner. Can you check?